xarray/tests/test_concat.py

```diff
@@ -23,6 +24,90 @@
     from xarray.core.types import CombineAttrsOptions, JoinOptions
 
 
+# helper method to create multiple tests datasets to concat
+def create_concat_datasets(
+    num_datasets: int = 2, seed: int | None = None, include_day: bool = True
+) -> list[Dataset]:
+    random.seed(seed)
+    result = []
+    lat = np.random.randn(1, 4)
+    lon = np.random.randn(1, 4)
```

Comment:
```suggestion
    rng = default_rng(seed)
    lat = rng.standard_normal(size=(1, 4))
    lon = rng.standard_normal(size=(1, 4))
    result = []
```
Not a big thing but [randn](https://numpy.org/doc/stable/reference/random/generated/numpy.random.randn.html#numpy-random-randn) is not really recommended practice anymore: https://numpy.org/doc/stable/reference/random/index.html#quick-start

Reply:
I'll take care of the other randn locations later.